### PR TITLE
Update docs for SA platform support

### PR DESCRIPTION
**Pull request by jcheng-splunk**

### CODE REVIEW

**Review by emaderer — APPROVED** [review]

File: internal/receiver/smartagentreceiver/README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+> **Note:** The SignalFx Smart Agent receiver is only supported on x86_64/amd64 platforms.
+
 The Smart Agent Receiver allows you to utilize existing [SignalFx Smart Agent monitors](https://github.com/signalfx/signalfx-agent#monitors)
 as OpenTelemetry Collector metric receivers.  It assumes that you have a properly configured environment with a
 functional [Smart Agent release bundle](https://github.com/signalfx/signalfx-agent/releases/latest) on your system,
-which is already provided for supported Splunk Distribution of OpenTelemetry Collector
+which is already provided for x86_64/amd64 Splunk Distribution of OpenTelemetry Collector
 [installation paths](https://github.com/signalfx/splunk-otel-collector#getting-started).
 
 **Beta: All Smart Agent monitors are supported at this time.  Configuration and behavior may change without notice.**
```
Comment (rmfitzpatrick) [inline internal/receiver/smartagentreceiver/README.md:11]:
```suggestion
**Beta: All Smart Agent monitors are supported at this time for their applicable platforms.  Configuration and behavior may change without notice.**
```

**Review by rmfitzpatrick — APPROVED** [review]
thanks

**Comment by jcheng-splunk** [comment]
cc @tmorrisonsfx

**Review by tmorrisonsfx — APPROVED** [review]
My only comment is to consider changing "Bundle" to "bundle", unless "collectd Bundle" is a proper noun. Thanks!

**Pull request merged**